api/src/main/java/com/alibaba/nacos/api/config/ConfigService.java

```diff
@@ -143,6 +144,14 @@ boolean publishConfigCas(String dataId, String group, String content, String cas
      * @return whether health
      */
     String getServerStatus();
+
+    /**
+     * add config filter.
+     * It is recommended to use {@link com.alibaba.nacos.api.config.filter.AbstractConfigFilter} to expand the filter.
+     *
+     * @param configFilter filter
+     */
```

Comment:
Add `since` comment to show when the api start to support.

Reply:
done

Reply:
maybe 2.3.0, the 2.2.1 has released and not include the api.